app/src/main/java/com/example/poste/api/poste/models/User.java

```diff
@@ -7,18 +7,24 @@
 import com.example.poste.api.poste.exceptions.MalformedResponseException;
 import com.example.poste.api.poste.exceptions.NoUserFoundException;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class User {
 
     private int id;
     private String email;
     private String username;
     private String password;
+    private List<Folder> folders;
```

Comment:
Hate for folks to duplicate effort, but these models have been rebuilt (with additional functionality used by the new LoginActivity and RegisterActivity pages) -- if you pull dev into your branch you should see these in the `com.example.poste.models` package

Reply:
Looks like my branch fell behind. I believe I have correct this now and gotten mine back in sync with current development.